apps/examples/helloxx/helloxx_main.cxx

```diff
@@ -148,6 +148,9 @@ extern "C"
 {
 	int helloxx_main(int argc, char *argv[])
 	{
+		// Print the cpp version used
+		printf("c++ version used : %d\n", __cplusplus);
```

Comment:
How about we add this log in `up_cxxinitialize` so that the CPP version is clearly displayed during boot up?

Reply:
__cplusplus can only be used in a cpp file. So, I think we can't add it there.